docs: migrating csi - add info to ensure that the cluster and storage solutions are health prior migrations (#955)
replicatedhq/kurl.sh: src/markdown-pages/install-with-kurl/migrating-csi.md

- The add-ons Kots and Registry require an Object Storage API (similar to AWS S3) to be available in the cluster. This API can be provided either by Rook (directly) or by the OpenEBS+Minio duo (OpenEBS provides the storage and Minio the Object Storage API). If you are using Kots or Registry add-ons in your cluster then the available migration paths are to migrate to Rook or to OpenEBS+Minio.

> - Ensure that cluster is a healthy state prior to perform any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy. To perform these checks see:

Reviewer: ```suggestion
- Ensure that cluster is in a healthy state prior to performing any migration. Check if the nodes are Ready and if all Pods are running. Also, if you are migrating from a storage solution to another please ensure that it is healthy. To perform these checks see:
```

> - Ensure that cluster is a healthy state prior do any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy prior the migration. Beyond check if all Pod(s) into this namespace are Running you might want check the storage state:

Author: ```suggestion
- Ensure that cluster is a healthy state prior to perform any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy. To do perform this checks see:
```

- The add-ons Kots and Registry require an Object Storage API (similar to AWS S3) to be available in the cluster. This API can be provided either by Rook (directly) or by the OpenEBS+Minio duo (OpenEBS provides the storage and Minio the Object Storage API). If you are using Kots or Registry add-ons in your cluster then the available migration paths are to migrate to Rook or to OpenEBS+Minio.

> - Ensure that cluster is a healthy state prior to perform any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy. To do perform this checks see:

Reviewer: ```suggestion
- Ensure that cluster is a healthy state prior to perform any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy. To perform these checks see:
```
